libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts

```diff
@@ -202,6 +204,16 @@ export interface IInvitationUserResponse {
     errorMessage?: string;
 }
 
+/**
+ * @internal
+ */
+export interface ICSPDirective {
+    id: string;
+    attributes: {
+        sources: Array<string>;
```

Comment:
Please could we add some docs? At least, what is this array of strings, as string can be almost anything...

Reply:
Added